src/test/scala/io/iohk/ethereum/rlp/RLPSuite.scala

```diff
@@ -387,14 +387,14 @@ class RLPSuite extends FunSuite
     val expected = "c88363617483646f67"
     val data = RLP.encode(RLPList("cat", "dog"))
     assert(expected == Hex.toHexString(data))
-    val dataObtained = decode[Seq[String]](data)
+    val dataObtained = decode[Seq[String]](data)(stringSeqEncDec)
```

Comment:
Interesting...
It also works if we move the implicit decoder to the top of the class. Just saying.. this is OK, too.